utils/stringutils/stringutils_test.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package stringutils
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestStringifyNilShouldReturnEmpty(t *testing.T) {
+	// given
+	var objects []interface{}
+	// when
+	strings := Stringify(objects)
+	// then
+	assert.Empty(t, strings)
+}
+
+func TestStringifyEmptyShouldReturnEmpty(t *testing.T) {
```

Comment:
nit: would it make sense to have a single `TestStringify` with subtests instead (similarly below)?

Reply:
Just to be sure, are you talking about something like that? https://ieftimov.com/post/testing-in-go-subtests/
One drawback I can see doing this is that subtests force to use only `assert.Equal` method, but I can get a try if you feel so.

Reply:
I propose a new patch using subtests. I may slightly prefer the previous version which is more TDD approach, one behavior / one function, but as you wish ;-)

Reply:
The approach you took is definitely valid, and I have no objections.

One other approach is just to think of subtests as a means of grouping, e.g.:
```
func TestStringify(t *testing.T) {
    t.Run("nil should return empty", func(t *testing.T) {
        // ...
    })
    t.Run("empty should return empty", func(t *testing.T) {
        // ...
    })

Reply:
I prefer the above when the amount of duplication is low (e.g. in this case, where the SUT is basically just two lines.) The other approach is great when there's a lot of complex setup and teardown that you'd like to avoid duplicating.